CONFIGURATION.md

```diff
@@ -16,19 +16,19 @@ Settings specific to `solarwinds_apm` are prefixed by `SW_APM_` and described in
 
 ### Exporter
 
-The default `solarwinds` exporter which communicates with the SolarWinds Observability backend is always configured. Additional exporters can be configured via the `OTEL_TRACES_EXPORTER` environment variable. For example, console exporter is part of standard installation and can be enabled via:
+The default `solarwinds` exporter which communicates with the SolarWinds Observability backend is always configured as OTLP Exporter. Additional exporters can be configured via the `OTEL_TRACES_EXPORTER` environment variable. For example, console exporter is part of standard installation and can be enabled via:
```

Comment:
Does setting `OTEL_TRACES_EXPORTER=console` or `OTEL_TRACES_EXPORTER=jaeger` stop export by OTLP?

Maybe to match this text (`Additional exporters`) to the examples below, we can update the examples to be:

```
export OTEL_TRACES_EXPORTER=otlp,console
export OTEL_TRACES_EXPORTER=otlp,jaeger
```

Reply:
no, it won't stop export by otlp. otlp will always be the default exporter; anything that set through `OTEL_TRACES_EXPORTER` will be additional exporter